packages/cli-template-monorepo-ethers/apps/web-app/src/components/PageContainer.tsx

```diff
@@ -35,15 +24,17 @@ export default function PageContainer({
     }
 
     return (
-        <div>
+        <>
             <div className="header">
-                <Link href="/" className="header-left">
-                    Feedback
-                </Link>
+                <div className="header-left">
```

Comment:
It can be useful to keep this link so that it's easier to go to the beginning of the app:

Then it would be:

```tsx
                <Link href="/" className="header-left">
                    Feedback
                </Link>
```

Instead of:

```tsx
                <div className="header-left">
                    {/* <Link href="/" className="header-left">
                        Feedback
                    </Link> */}
                </div>
```

Then `import Link from "next/link"` should be added to the imports.

I know this is not part of the current boilerplate, but I think it's useful to keep it.